src/mscorlib/src/System/AggregateException.cs

```diff
@@ -474,23 +474,6 @@ public override string ToString()
 
             return text;
         }
-
-        /// <summary>
-        /// This helper property is used by the DebuggerDisplay.
-        /// 
-        /// Note that we don't want to remove this property and change the debugger display to {InnerExceptions.Count} 
-        /// because DebuggerDisplay should be a single property access or parameterless method call, so that the debugger 
-        /// can use a fast path without using the expression evaluator.
-        /// 
-        /// See http://msdn.microsoft.com/en-us/library/x810d419.aspx
-        /// </summary>
-        private int InnerExceptionCount
```

Comment:
Thanks.  This is also something it'd be good to have some tests in CoreFx for.  We have them in a few libraries, with tests that look at types we expect to have debugger attributes, and then use some basic parsing and reflection to verify that the expected members exist and can be invoked.  But it'd be good to effectively cover all of our types with such attributes; it's so easy to accidentally remove such members, either manually or via automatic tools like this.